src/test/java/com/solvd/eshop/ConfigFileReader.java

```diff
@@ -0,0 +1,18 @@
+package com.solvd.eshop;
+
+import java.io.*;
+import java.util.Properties;
+
+public class ConfigFileReader {
+    public static String getData(String propertyKey) {
+        Properties properties = new Properties();
+        try {
+            InputStream in = new BufferedInputStream(new FileInputStream("src/main/resources/config.properties"));
+            properties.load(in);
+            in.close();
+            return properties.getProperty(propertyKey);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+}
```

Comment:
It's better to create different pages under main: pages(for implemented pages) and utils (for classes like this), and move this file to utils package. Now it's looks not important, but for big projects, it's difficult to find necessary information if classes are not sorted